src/components/widgets/filesystem/FileSystem.vue

```diff
@@ -370,6 +405,38 @@ export default class FileSystem extends Mixins(StateMixin, FilesMixin, ServicesM
     return this.$store.state.server.info.registered_directories || []
   }
 
+  transformTimelapseItems (items: AppFile[] | AppDirectory[]) {
+    const timelapses: {[key: string]: AppFile} = {}
+
+    for (const item of items) {
+      if (item.type === 'file' && item.extension !== 'jpg') {
+        timelapses[item.filename.slice(0, -(item.extension.length + 2))] = item
```

Comment:
Should be `slice(0, -(item.extension.length + 1))`.

![image](https://user-images.githubusercontent.com/2041996/160261717-49b778b4-f2a5-477c-a59d-42afebe0953d.png)